src/singletons/Settings.hpp

```diff
@@ -17,6 +17,8 @@ using TimeoutButton = std::pair<QString, int>;
 
 namespace chatterino {
 
+enum TrayIconTrigger { OnMinimize, OnClose, OnCloseAndMinimize };
+
```

Comment:
I'd move this enum to a different file, similar to how it is done with `StreamerModeSetting`. Perhaps `Window.hpp` might be a good place for it.